.travis.yml

```diff
@@ -55,6 +55,9 @@ addons:
       - texlive-luatex
       - texlive-xetex
       - ttf-wqy-zenhei
+      - xvfb
```

Comment:
Do we have to change these things for all of the envs?

Reply:
No, this is required for only arm64 platform as xvfb is not getting started in aarch64 platform through "services". Updated the same.